src/styles/fonts.tsx

```diff
@@ -40,8 +41,110 @@ const standards = {
     color: colors.dark,
   },
 }
-// Figma Font Styles
-const fontStyles = StyleSheet.create({
+
+/**
+ * Figma TypeScale Styles
+ */
+export const typeScale = StyleSheet.create({
+  displayLarge: {
+    fontFamily: Inter.Bold,
+    fontSize: 80,
+    lineHeight: 80,
+    letterSpacing: -2.4,
+  },
+  displayMedium: {
+    fontFamily: Inter.Bold,
+    fontSize: 56,
+    lineHeight: 64,
+    letterSpacing: -1.12,
+  },
+  displaySmall: {
+    fontFamily: Inter.Bold,
+    fontSize: 40,
+    lineHeight: 48,
+    letterSpacing: -0.8,
+  },
+  titleLarge: {
+    fontFamily: Inter.Bold,
+    fontSize: 32,
+    lineHeight: 40,
+    letterSpacing: -0.32,
+  },
+  titleMedium: {
+    fontFamily: Inter.Bold,
+    fontSize: 24,
+    lineHeight: 32,
+  },
+  titleSmall: {
+    fontFamily: Inter.Bold,
+    fontSize: 20,
+    lineHeight: 28,
+  },
+  labelLarge: {
+    fontFamily: Inter.Bold,
+    fontSize: 18,
+    lineHeight: 28,
+  },
+  labelMedium: {
+    fontFamily: Inter.Medium,
+    fontSize: 16,
+    lineHeight: 24,
+  },
+  labelSmall: {
+    fontFamily: Inter.Medium,
+    fontSize: 14,
+    lineHeight: 20,
+  },
+  labelXSmall: {
+    fontFamily: Inter.Medium,
+    fontSize: 12,
+    lineHeight: 16,
+    letterSpacing: 0.16,
```

Comment:
should this be 0.12? Seems like everything else is a % of `fontSize`, except this which seems to be a % of `lineHeight`

Reply:
I think it should be too on second glance based on what I see in Figma: % appears related to the fontSize instead lineHeight. Fixed in https://github.com/valora-inc/wallet/pull/4211/commits/eae1ba9ac015d041d7168da7b4301c6ca297b0cf.